src/factory/wrapper.js

```diff
@@ -5,11 +5,17 @@ export default ($ad, options) => {
   const wrapper = new Wrapper()
   const $wrapper = $ad.wrapper
   inheritAd($ad, $wrapper, wrapper, options)
+  if ($wrapper.followAdditionalWrappers != null) {
```

Comment:
Why are we changing this?

Reply:
followAdditionalWrappers has a default value of true, and allowMultipleAds has a default value of false in the VAST 4.0 spec. So we cannot simply do the assignment.